Example/AccessibilitySnapshot/SwiftUIView+EmbedInHostingController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SwiftUIView+EmbedInHostingController.swift
+//  AccessibilitySnapshotDemo
+//
+//  Created by Felizia Bernutz on 15.10.20.
+//  Copyright © 2020 CocoaPods. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+
+@available(iOS 13.0, *)
+extension View {
```

Comment:
This is definitely an acceptable solution - but it may also we be worth us adding SwiftUI support to the SnapshotTesting spec at some point!

The base library does support it so would remove this extension being necessary for us or users of the library!

Reply:
yes definitely, would be great 👍

Reply:
I've added a comment to remember to remove this extension again when #37 is fixed and snapshotting SwiftUI views is supported. Hope that's fine for the moment.